backend/spotihue/oauth.py

```diff
@@ -0,0 +1,84 @@
+import logging
```

Comment:
Do we need doc strings/type hints here?

Reply:
good call! Forgot about that; I'll add em